std/http/file_server.ts

```diff
@@ -18,16 +18,18 @@ interface EntryInfo {
   name: string;
 }
 
-interface FileServerArgs {
+export interface FileServerArgs {
   _: string[];
   // -p --port
-  p: number;
-  port: number;
+  p?: number;
+  port?: number;
   // --cors
-  cors: boolean;
+  cors?: boolean;
+  // -d
+  d?: "false";
```

Comment:
Renaming this to `--no-dir-listing` would make it very clear that it is enabled by default. It can be a boolean then. WDYT?